docs/scos/dev/front-end-development/202212.0/oryx/building-components/component-definition.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Oryx Components Definition"
+description: Components are registered in an Oryx application by a definition file
+last_updated: Sept 19, 2023
+template: concept-topic-template
+---
+
+Oryx components can be integrated in different ways. Components can be listed in [pages](/docs/scos/dev/front-end-development/{{page.version}}/oryx/building-pages/oryx-pages.html) and [compositions](/docs/scos/dev/front-end-development/{{page.version}}/oryx/building-pages/oryx-compositions.html) or can be used inside other components or content integrated from a 3rd party CMS.
+
+When a component is rendered for the first time, Oryx resolves the component definition from the registry and loads the associated implementation. With this, components are lazily loaded.
+
+## Create a component definition
+
+To register a [component implementation](/docs/scos/dev/front-end-development/{{page.version}}/oryx/building-components/component-implementation.html), you need to provide a component definition. The component definition requires a name and implementation. The name is used as the (web) component element name. Web components require as least to be defined by 2 words, separated by a dash. It is a good practice to prefix your components by a vendor prefix, such as the project, brand or company name. This is why Oryx components are all prefixed with `oryx-`.
+
+{% info_block infoBox %}
```

Comment:
Formatting leftover?

Reply:
The info block? No that's deliberately, to highlight something.